tests/rados/test_stretch_mon_replacements.py

```diff
@@ -0,0 +1,263 @@
+
+    # Checking which DC mon to be replaced, It would be either data site or Arbiter site
+    # Proceeding to replace mon in either one of the Data DCs if crush name sent is either DC1 or DC2.
+    if replacement_site in [dc_1_name, dc_2_name]:
+        log.debug(f"Proceeding to replace mon of the data site {dc_1_name}")
+
+        # Removing mon service
+        if not mon_obj.remove_mon_service(host=mon_host.hostname):
+            log.error("Could not remove the Data site mon daemon")
+            raise Exception("mon service not removed error")
+
+        quorum = mon_obj.get_mon_quorum_hosts()
+        if mon_host.hostname in quorum:
+            log.error(
+                f"selected host : {mon_host.hostname} mon is present as part of Quorum post removal"
+            )
+            raise Exception("Mon in quorum error post removal")
+
+        # Adding the mon back to the cluster with crush locations
+        if not mon_obj.add_mon_service(
+            host=mon_host.hostname,
+            ip=mon_host.ip_address,
+            location_type="datacenter",
+            location_name=dc_1_name,
+        ):
+            log.error("Could not add data site mon service")
+            raise Exception("mon service not added error")
+
+        quorum = mon_obj.get_mon_quorum_hosts()
+        if mon_host.hostname not in quorum:
+            log.error(
+                f"selected host : {mon_host.hostname} mon is not present as part of Quorum post addition"
+            )
+            raise Exception("Mon not in quorum error post addition")
+
+        log.info("A data site mon was removed and added back. Pass")
+
+    else:
+        log.info("replacing Arbiter Mon daemon")
+
+        found = False
+        # Arbiter mon cannot be removed directly. A new tiebreaker mon should be deployed temporarily
+        for item in cluster_nodes:
+            if item.hostname not in [mon.hostname for mon in mon_nodes]:
+                alt_mon_host = item
+                found = True
+                break
+        if not found:
+            log.error("No alternate mon host was found... fail")
+            raise Exception("No temp mon host found error")
+
+        log.debug(
+            f"Selected temp mon host: {alt_mon_host.hostname} with "
+            f"IP : {alt_mon_host.ip_address} to be deployed for replacement of Arbiter mon"
+        )
+
+        # making sure that no mon daemon is running on the alternate host selected.
+        quorum = mon_obj.get_mon_quorum_hosts()
+        if alt_mon_host.hostname in quorum:
+            log.error(
+                f"selected alternate host : {alt_mon_host.hostname} has already a mon daemon present as part of Quorum"
+            )
+            raise Exception("Mon daemon already present error")
+
+        log.debug(
+            "No mon daemon present on selected host. Proceeding to deploy temp tiebreaker mon"
+        )
+        # Adding the mon back to the cluster
```

Comment:
please remodify the comment here by removing "back to the cluster" which implies mon was already present, removed and adding back to cluster.

Reply:
done.